web/server.js

```diff
@@ -137,7 +137,7 @@ app.use('/', (req, res) => {
   const isTmrowCo = req.get('host').indexOf('electricitymap.tmrow') !== -1;
   const isNonWWW = req.get('host') === 'electricitymap.org'
     || req.get('host') === 'live.electricitymap.org';
-  const isStaging = req.get('host') === 'staging.electricitymap.org';
+  const isStaging = req.get('host').includes('staging');
```

Comment:
Required to make sure our staging environment is better detected (when e.g. called by cloud run)